app/api/events.py

```diff
@@ -104,6 +105,14 @@ class Meta:
                                  schema='EventInvoiceSchema',
                                  many=True,
                                  type_='event_invoice')
+    discount_codes = Relationship(attribute='discount_code',
```

Comment:
`discount_code_id` is a foreign key, afaik removing that would not remove the relationship